java/src/jmri/jmrit/display/layoutEditor/LayoutDoubleXOver.java

```diff
@@ -112,10 +115,32 @@ public LayoutDoubleXOver(@Nonnull String id,
      * @param v version, unused.
      */
     public LayoutDoubleXOver(@Nonnull String id, 
-            @Nonnull LayoutEditor layoutEditor,
+            @Nonnull LayoutEditor layoutEditor, 
             int v) {
-        super(id, TurnoutType.DOUBLE_XOVER, layoutEditor, v);
+        super(id, TurnoutType.DOUBLE_XOVER, layoutEditor, 1);
+    }
+
+    @Override
+    @Nonnull
+    protected List<String> getDiagonalPaths() {
+        return Arrays.asList("AC", "BD"); // Both diagonals
+    }
+
+    @Override
+    protected String findPathForAnchor(@Nonnull String anchor) {
+        switch (anchor) {
+            case "A":
+                return "AB"; // Could also be "AC" depending on turnout state
+            case "B":
+                return "AB"; // Could also be "BD" depending on turnout state
+            case "C":
+                return "CD"; // Could also be "AC" depending on turnout state
+            case "D":
+                return "CD"; // Could also be "BD" depending on turnout state
+            default:
+                return null;
+        }
     }
     
-    // private final static org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(LayoutDoubleXOver.class);
+    // private final static org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(LayoutXOver.class);
```

Comment:
Although it's commented out, it would be good to have this logging definition statement using the proper class name.  Perhaps this was an inadvertent change?